src/Administration/Resources/app/administration/src/core/service/login.service.js

```diff
@@ -321,9 +321,9 @@ export default function createLoginService(httpClient, context, bearerAuth = nul
      * @returns {CookieStorage}
      */
     function cookieStorageFactory() {
-        const domain = context.host;
+        // take current domain instead of context to enable other hosts than "localhost"
+        const domain = window.location.hostname;
```

Comment:
login.service.js?3aeb:324 Uncaught ReferenceError: window is not defined
    at cookieStorageFactory (login.service.js?3aeb:324)
    at createLoginService (login.service.js?3aeb:23)
    at onMessage (admin-worker.worker.js?5b99:20)

It needs to work also without `window` object

Reply:
Thanks for the feedback - its just a quick hack to demonstrate the problem. I don't know if its on purpose that the watcher process is restricted to "localhost" but at least for my development process, it needs to work with custom dev domains as well.